### DISCUSSION TITLE
unit_conv_cap_to_flow argument in model

### DISCUSSION THREAD

**Discussion by asashishh** [spine-tools/SpineOpt.jl#1067] [Support]
Hi, 
I tried to implement the start_up_limit parameter in the energy system model that I created. I get the following error on execution of model

LoadError: can't find a value of unit_conv_cap_to_flow for argument(s) (unit = electrolyser, node = elec_dc, direction = from_node, stochastic_scenario = realization, analysis_time = 2000-12-30T00:00, t = 2000-12-30T00:00~>2000-12-30T00:10)

Although the documentation mentions this parameter but I don't find this parameter for unit connection in the model. Similar parameter is available for connection.

SpineOpt: 0.6.16
SpineInterface: 0.11.15
Julia: 1.8.5

**Comment by tarskul** [comment]
It is hard to tell what the issue is exactly. Especially since you seem to be using a rather old version of SpineOpt and SpineInterface (in the latest version I do see these parameters in the unit connection). Regardless, unit_conv_cap_to_flow is optional and has a default value of 1.0 so it would be strange that it would be the cause of the issue. Maybe there is another parameter missing and the warning message simply runs into an error first on this parameter? E.g. is the unit capacity defined?

You may want to look at the constraints where these parameters are used. That may give a clue as to what parameters exactly you may be missing: https://spine-tools.github.io/SpineOpt.jl/latest/mathematical_formulation/constraints_automatically_generated/#constraint_unit_flow_capacity
